emma-sax4/emma-sax4.github.io
# Overlay y axis scrollbar

## Changes
* Overlay the Y axis scrollbar so that the page doesn't move to the left a little bit when the right-hand scrollbar appears

## QA
As a quality check, make sure that this solution is verified on Chrome, Safari, and Firefox (the only browsers that are officially "supported" by this site. Also, verify that the scrolling functionality is the same as before in all browser options.
